2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_setup.py:_flush():71] setting env: {}
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_163502-1mh5y3qg/logs/debug.log
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_163502-1mh5y3qg/logs/debug-internal.log
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1a8762f100>
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_init.py:init():404] calling init triggers
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-10 16:35:02,162 INFO    MainThread:592949 [wandb_init.py:init():460] starting backend
2022-02-10 16:35:02,162 INFO    MainThread:592949 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-10 16:35:02,164 INFO    MainThread:592949 [backend.py:ensure_launched():216] starting backend process...
2022-02-10 16:35:02,175 INFO    MainThread:592949 [backend.py:ensure_launched():221] started backend process with pid: 609506
2022-02-10 16:35:02,176 INFO    MainThread:592949 [wandb_init.py:init():469] backend started and connected
2022-02-10 16:35:02,189 INFO    MainThread:592949 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-10 16:35:02,189 INFO    MainThread:592949 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-10 16:35:02,189 INFO    MainThread:592949 [wandb_init.py:init():533] updated telemetry
2022-02-10 16:35:02,212 INFO    MainThread:592949 [wandb_init.py:init():563] communicating current version
2022-02-10 16:35:02,904 INFO    MainThread:592949 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-10 16:35:02,905 INFO    MainThread:592949 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-10 16:35:03,191 INFO    MainThread:592949 [wandb_init.py:init():606] starting run threads in backend
2022-02-10 16:35:05,261 INFO    MainThread:592949 [wandb_run.py:_console_start():1810] atexit reg
2022-02-10 16:35:05,261 INFO    MainThread:592949 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-10 16:35:05,261 INFO    MainThread:592949 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-10 16:35:05,262 INFO    MainThread:592949 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-10 16:35:05,262 INFO    MainThread:592949 [wandb_init.py:init():633] run started, returning control to user process
2022-02-10 16:35:05,262 INFO    MainThread:592949 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-10 16:35:05,286 INFO    MainThread:592949 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-10 16:35:05,286 INFO    MainThread:592949 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-10 16:36:26,858 INFO    MainThread:592949 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_setup.py:_flush():71] setting env: {}
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_163626-5wbp7itl/logs/debug.log
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_163626-5wbp7itl/logs/debug-internal.log
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_init.py:init():404] calling init triggers
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-10 16:36:26,866 INFO    MainThread:592949 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 1mh5y3qg
2022-02-10 16:36:26,868 INFO    MainThread:592949 [wandb_run.py:finish():1457] finishing run nelsonsw5/3DDet_Viz/1mh5y3qg
2022-02-10 16:36:26,868 INFO    MainThread:592949 [jupyter.py:save_history():429] not saving jupyter history
2022-02-10 16:36:26,868 INFO    MainThread:592949 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-10 16:36:26,868 INFO    MainThread:592949 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-10 16:36:26,868 INFO    MainThread:592949 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-10 16:36:26,869 INFO    MainThread:592949 [wandb_run.py:_restore():1752] restore
2022-02-10 16:36:27,851 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-10 16:36:28,053 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-10 16:36:28,216 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11615
}

2022-02-10 16:36:28,319 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-10 16:36:28,421 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8253
  total_bytes: 11641
}

2022-02-10 16:36:28,523 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-10 16:36:28,625 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-10 16:36:28,727 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-10 16:36:28,828 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-10 16:36:28,929 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-10 16:36:29,084 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-10 16:36:29,338 INFO    MainThread:592949 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}
local_info {
}

2022-02-10 16:36:30,473 INFO    MainThread:592949 [wandb_run.py:_append_files():2180] logging synced files
